### PR TITLE
Htmltask

### PR DESCRIPTION
**Pull request by Verdantcoderr**

### CODE REVIEW

File: submissions/OpemipoBanso/HTML/kasuwa.com-registration page.html [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+                        <input type="email-address" name="email-address" id="email-address" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}"> <br>
+    
+    
+                        <label for="Phone-number">Phone :</label> <br>
+                            <select>
+                                    <!-- Countries often selected by users can be moved to the top of the list. -->
+                                    <!-- Countries known to be subject to general US embargo are commented out by default. -->
+                                    <!-- The data-countryCode attribute is populated with ISO country code, and value is int'l calling code. -->
+    
+                                    <option data-countryCode="NG" value="234">Nigeria (+234)</option>
```
Comment (adaugochi) [inline submissions/OpemipoBanso/HTML/kasuwa.com-registration page.html:57]: Well done. Although there are better ways of loading the list of countries in a form page

**Review by adaugochi — APPROVED** [review]

**Comment by Verdantcoderr** [comment]
I would love to be intimated with ways to generate the country codes as the
procedure I used consumed a whole lot of space

I also added the title attribute to guide users through creating passwords
but was not displayed... Could there be any reason for this?

Warm regards

**Comment by adaugochi** [comment]
Send me a DM on slack (Adaa) so we can talk more

**Review by tegaphilip — APPROVED** [review]
Good job @Verdantcoderr . I will approve this but will advise that you avoid using spaces in your file names. the `docx` file and `jpg` files have spaces in them. In subsequent tasks, please avoid this as they might cause funny errors when you least expect.

Otherwise, good job. Do not forget to contact @adaugochi on how to automatically populate the countries and their codes so that you can have a leaner html file.

**Pull request merged**